src/components/markdown/references/references.jsx

```diff
@@ -0,0 +1,214 @@
+            </p>
+          </div>
+        )
+      }
+    }
+
+    // Chicago
+    else if(citationStyle == "Chicago")
+    {
+      if(types == "book")
+      {
+        return (
+          <div id={`#reference-${referenceId}`}>
+            <p>
+              {' '}
+              {i + 1}. {authorName[1]}, {authorName[0]}. {fields.year}. <i>{fields.title}</i>. {fields.address}: {fields.publisher}.
+            </p>
+          </div>
+        )
+      }
+      else if(types == "article")
+      {
+        return (
+          <div id={`#reference-${referenceId}`}>
+            <p>
+              {' '}
+              {i + 1}. {authorName[1]}, {authorName[0]}. {fields.year}. "{fields.title}". <a target='_blank' href={fields.url}>{fields.url}</a>.
+            </p>
+          </div>
+        )
+      }
+    }
+
+    // IEEE
+    else if(citationStyle == "IEEE")
+    {
+      if(types == "book")
+      {
+        return (
+          <div id={`#reference-${referenceId}`}>
+            <p>
+              {' '}
+              [{i + 1}]{authorName[0].charAt(0)}. {authorName[1]}, <i>{fields.title}</i>. {fields.address}: {fields.publisher}, {fields.year}.
+            </p>
+          </div>
+        )
+      }
+      else if(types == "article")
+      {
+        return (
+          <div id={`#reference-${referenceId}`}>
+            <p>
+              {' '}
+              [{i + 1}]{authorName[0].charAt(0)}. {authorName[1]}, "{fields.title}", {fields.year}. [Online]. Available: <a target='_blank' href={fields.url}>{fields.url}</a>.
+            </p>
+          </div>
+        )
+      }
+    }
+
+    // AMA
+    else if(citationStyle == "AMA")
+    {
+      if(types == "book")
+      {
+        return (
+          <div id={`#reference-${referenceId}`}>
+            <p>
+              {' '}
+              {i + 1}. {authorName[1]} {authorName[0].charAt(0)}, <i>{fields.title}</i>. {fields.address}: {fields.publisher}, {fields.year}.
+            </p>
+          </div>
+        )
+      }
+      else if(types == "article")
+      {
+        return (
+          <div id={`#reference-${referenceId}`}>
+            <p>
+              {' '}
+              {i + 1}. {authorName[1]} {authorName[0].charAt(0)}. <a target='_blank' href={fields.url}>{fields.url}</a>. {fields.title}. Published {fields.year}.
+            </p>
+          </div>
+        )
+      }
+    }
+
+    // No citation style specified
+    else
+    {
+      return (
+        <div id={`#reference-${referenceId}`}>
+          <p>
+            {' '}
+            {i + 1}. {fields.author} ({fields.year}) - <i>{fields.title}</i> -{' '}
+            <a target='_blank' href={fields.url}>
+              {fields.url}
+            </a>
+          </p>
+        </div>
+      )
+    }
+  })
+
+  // return the map
+  return <div>{referenceOptions}</div>
+}
+
+References.PropTypes = {
+  children: PropTypes.isRequired,
+}
```

Comment:
These won't be needed once the `children` and `citationStyle` props are removed.

```suggestion
```